src/controller/python/OpCredsBinding.cpp

```diff
@@ -369,11 +373,74 @@ void pychip_OnCommissioningStatusUpdate(chip::PeerId peerId, chip::Controller::C
     return sTestCommissioner.OnCommissioningStatusUpdate(peerId, stageCompleted, err);
 }
 
+/**
+ * Call of pychip_OpCreds_AllocateControllerFoCustomCommissioningFlow allows allocating a device commissioner that will perform
+ * commissioning without using the auto-commissioning flow present in the Controller class
+ *
+ */
+PyChipError pychip_OpCreds_AllocateControllerForPythonCommissioningFLow(chip::Controller::DeviceCommissioner ** outDevCtrl,
```

Comment:
These methods take a lot of arguments and some seem to be representable by byte arrays (i.e. they are data + length pointers). 

Could we pass in a @dataclass containing the information needed and not have a method with 9 parameters?

Could be a followup TODO, however anything with more than 4 arguments is very hard to maintain.

Reply:
https://github.com/project-chip/connectedhomeip/issues/25214

Already working on this.